src/components/Image.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useImage } from 'use-cloudinary';
+
+// Now we can pass Cloudinary options & easily apply Chakra style props to our image
+import { Image as ChakraImage } from '@chakra-ui/core';
+
+export default function Image(props) {
+  const { cloudName, transforms, publicId } = props;
+
+  const { getImage, data, status, error } = useImage({ cloud_name: cloudName })
+
+  React.useEffect(() => {
+    getImage({
+      public_id: publicId,
+      transform_options: {
+        ...transforms
+      }
+    })
+  }, [])
+
+
+  // Can make use of the `Spinner` or `Skeleton` component from Chakra UI here 
```

Comment:
Mind turning these into @TODO comments? I'll comb the codebase periodically for them and make sure they're added to Trello.

Reply:
Most of my comments here were specifically for review purposes since the lib is fairly new. 

I'd be happy to do that though and make them actionable. Could you show me an example of what you're looking for?

Reply:
Oh that's rad. What the extension 👀

Reply:
https://marketplace.visualstudio.com/items?itemName=wayou.vscode-todo-highlight

It's got a nice little setting where you can highlight any keyword you'd like 😄